sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3032,6 +3032,17 @@ object GpuOverrides extends Logging {
       (a, conf, p, r) => new ComplexTypeMergingExprMeta[Concat](a, conf, p, r) {
         override def convertToGpu(child: Seq[Expression]): GpuExpression = GpuConcat(child)
       }),
+    expr[Conv](
+      desc = "Convert string representing a number from one base to another",
+      pluginChecks = ExprChecks.projectOnly(
+        outputCheck = TypeSig.STRING,
+        paramCheck = Seq(
+          ParamCheck(name = "num", cudf = TypeSig.STRING, spark = TypeSig.STRING),
+          ParamCheck(name = "from_base", cudf = TypeSig.integral, spark = TypeSig.integral),
+          ParamCheck(name = "to_base",   cudf = TypeSig.integral, spark = TypeSig.integral)),
```

Comment:
I think we need to tag this as partial support because Spark supports all numbers from 2 to 36 inclusive. We also should tag ours as needing from_base and to_base to be literal values.